CAIPs/caip-74.md

```diff
@@ -1,36 +1,34 @@
 ---
 caip: 74
-title: CACAO: Chain Agnostic CApability Object
-author: Sergey Ukustov (@ukstv)
+title: CACAO - Chain Agnostic CApability Object
+author: Sergey Ukustov (@ukstv), Haardik (@haardikk21)
 discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/74
 status: Draft
 type: Standard
 created: 2021-11-01
-updated: 2022-03-10
+updated: 2022-07-12
 ---
 
 ## Simple Summary
-
-Represent a chain-agnostic Object Capability (OCAP), created using [EIP-4361](https://github.com/ethereum/EIPs/blob/5e9b0fe0728e160f56dd1e4cbf7dc0a0b1772f82/EIPS/eip-4361.md) (or similar for other blockchains), as an [IPLD](https://ipld.io) object.
+Represent a chain-agnostic Object Capability (OCAP), created using [CAIP-122](), as an [IPLD](https://ipld.io) object.
 
 ## Abstract
-
-In this document we define a way to present a result of [EIP-4361 "Sign-in with Ethereum"](https://github.com/ethereum/EIPs/blob/5e9b0fe0728e160f56dd1e4cbf7dc0a0b1772f82/EIPS/eip-4361.md)
-signing operation as an [IPLD](https://ipld.io)-based object capability (OCAP).
-As we expect other blockchains to follow a path similar to Ethereum, CAIP seems to be the best place for such a proposal.
+In this document we define a way to present a result of [CAIP-122]() signing operation as an [IPLD](https://ipld.io)-based object capability (OCAP).
```

Comment:
```suggestion
In this document we define a way to record the result of [CAIP-122]() signing operation as an [IPLD](https://ipld.io)-based object capability (OCAP). This creates not just an event receipt of an authentication, but also a composable and replay-able authorization receipt for verifiable authorizations, when the message signed contains the appropriate fields. The first CACAO profile was tailored to the ethereum dapps supporting [EIP-4361][] but roughly equivalent profiles for other wallet/dapp ecosystems are being added over time.
```

Removing too many references to EIP-4361 was making it a little confusing, so I thought the abstract would be the right section to put back in the idea of "profiles" and 4361 as first instance of an abstract profiling process.